deploy-board/deploy_board/webapp/error_views.py

```diff
@@ -46,7 +47,11 @@ def process_exception(self, request, exception):
                 request.session.flush()
                 return HttpResponseRedirect("/")
 
+            debug_mode = os.environ.get("DEBUG_MODE", "OFF")
```

Comment:
Should convert it to a setting. i.e. https://github.com/pinterest/teletraan/blob/24b15b24491d4cbd429e5cd48a782e4d90b1539e/deploy-board/deploy_board/settings.py#L67